script2.js

```diff
@@ -0,0 +1,170 @@
+let age = prompt("How old are you?");
+if (age >= 0 && age < 2) alert("You are a child");
+if (age >= 12 && age < 18) alert("You are a teenager");
+if (age >= 18 && age < 60) alert("You are an adult");
+if (age >= 60 && age < 100) alert("You are a pensioner");
+
+let num = prompt("Choose number from 0 to 9");
+switch (num) {
+  case "0":
+    alert(")");
+    break;
+  case "1":
+    alert("!");
+    break;
+  case "2":
+    alert("@");
+    break;
+  case "3":
+    alert("#");
+    break;
+  case "4":
+    alert("$");
+    break;
+  case "5":
+    alert("%");
+    break;
+  case "6":
+    alert("^");
+    break;
+  case "7":
+    alert("&");
+    break;
+  case "8":
+    alert("*");
+    break;
+  case "9":
+    alert("(");
+    break;
+}
+
+let number = prompt("Enter 3 digit number");
+number = number + "";
+if (
+  number[0] == number[1] ||
+  number[0] == number[2] ||
+  number[1] == number[2]
+) {
+  alert("TRUE");
+} else alert("FALSE");
+
+let year = prompt("Enter year you want to check");
+if (year % 400 == 0 || (year % 4 == 0 && year % 100 != 0)) {

```

Comment:
лучше всегда использовать строгое равенство `===`
```suggestion
if (year % 400 === 0 || (year % 4 === 0 && year % 100 !== 0)) {

```